pages/buildkite_compute/macos_instances.md

```diff
@@ -0,0 +1,132 @@
+## Mac compute instances
+
+Mac instances are only offered with Mac silicon architecture. Please contact support if you have specific needs for Intel machines.
```

Comment:
Should this be "Mac" or "macOS"?

Also, I believe the term is [Apple silicon](https://en.wikipedia.org/wiki/Apple_silicon) and not "Mac silicon".

```suggestion
Mac instances are only offered with Apple silicon architecture. Please contact support if you have specific needs for Intel machines.
```